### PR TITLE
Cgi

### PR DESCRIPTION
**Pull request by vladkorsik**
1 Run the manual changes file
2 Run LS
3 Run Genric update (CGI =1)

### CODE REVIEW

**Review by Alexdavv — CHANGES_REQUESTED** [review]
1. readme file is missing.
2. We need to use the same CASE SQL syntaxis.
3. Sort out the sources/schema. Now one of them is not used.
4. Comments across the text.
5. The same additional issues as in civic.

File: CGI/create_source_tables.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    info text,
+    context varchar(255),
+    cancer_acronym varchar(255),
+    source varchar(255),
+    reference text
+);
+
+
+-- with existing sources
+DROP TABLE IF EXISTS dev_cgi.genomic_cgi_backup210727;
```
Comment (hardhouse) [inline CGI/create_source_tables.sql:17]: why it is here? it should be in "manual_work" folder

File: CGI/load_stage.sql [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+**************************************************************************/
+
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'CGI',
+	pVocabularyDate			=> current_date,
+	pVocabularyVersion		=> 'CGI'||TO_CHAR(CURRENT_DATE,'YYYYMMDD'),
+	pVocabularyDevSchema	=> 'dev_cgi',
+	pAppendVocabulary		=> FALSE
```
Comment (hardhouse) [inline CGI/load_stage.sql:27]: you can omit this clause,  because one vocabulary

**Review by hardhouse — CHANGES_REQUESTED** [review]

**Comment by hardhouse** [comment]
+ same comments as in https://github.com/OHDSI/Vocabulary-v5.0/pull/688

**Comment by vladkorsik** [comment]
@Alexdavv 25 from 37 of invalid cocnepts shuold be Invalidated with no possibility to Update them somehow (because of thier ambiguity, as they are protein level alteratations related to several gen level SNVs.
12 from 37 are for some reason exist in so-called 2021 source, but the official distributive (2018) can not proove it.
 The current approach to rebuild the CGI is to 
 Run manual_changes 2022-10-04 to set correct names and concept_codes 
Run LS and Generic Update 
 Run manual_changes 2022-10-07 to set valid_end_dates

File: CGI/load_stage.sql (hunk 2 above)
Comment (vladkorsik) [reply CGI/load_stage.sql]: done

File: CGI/create_source_tables.sql (hunk 1 above)
Comment (vladkorsik) [reply CGI/create_source_tables.sql]: resolved

File: CGI/load_stage.sql [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+* Authors: Varvara Savitskaya, Vlad Korsik, Alexander Davydov
+* Date: Fall-2022
+**************************************************************************/
+
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'CGI',
+	pVocabularyDate			=> current_date,
+	pVocabularyVersion		=> 'CGI'||TO_CHAR(CURRENT_DATE,'YYYYMMDD'),
```
[the remark this answers is not in the document]
Comment (vladkorsik) [reply CGI/load_stage.sql]: corrected

**Review by hardhouse — APPROVED** [review]

**Review by Alexdavv — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

**Pull request merged**